backend/dev/user.clj

```diff
@@ -27,6 +27,7 @@
    [app.common.transit :as t]
    [app.common.types.file :as ctf]
    [app.common.uuid :as uuid]
+   [app.common.uri :as u]
```

Comment:
This requirement (and most on this file) seems to be unused

Reply:
This is a user namespace, is used for dev and having this exported is useful